pkgs/applications/misc/double-commander/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  stdenv, lib
+  , fetchFromGitHub
+  , fpc, lazarus
+  , which, autoPatchelfHook
+  , libqt5pas, qt5, gtk2, dbus
+}:
+
+let qtLazarus = lazarus.override {
```

Comment:
I just wonder why you need to override lazarus? I maintain [cudatext](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/cudatext/default.nix), that's lazarus project as well.

Reply:
I guess it's for Lazarus IDE, but we don't need IDE here, we use only lazbuild.

Reply:
From all-packages.nix:
```
  lazarus = callPackage ../development/compilers/fpc/lazarus.nix {
    fpc = fpc;
  };

  lazarus-qt = libsForQt5.callPackage ../development/compilers/fpc/lazarus.nix {
    fpc = fpc;
    withQt = true;
  };
```

Reply:
So we can use lazarus or lazarus-qt, there is no difference for us. Both provide lazbuild.